README.md

```diff
@@ -4,7 +4,6 @@
 
 > Description the project.
 
-
 ## Built With
 
```

Comment:
- [ ] Kindly make sure that you're README.md file is updated and edited with the proper information to better suit your project.